CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+# Code of Conduct
+
+This project follows the 'alphagov' Github organisation's Code of Conduct (https://github.com/alphagov/.github/blob/master/CODE_OF_CONDUCT.md).
```

Comment:
GitHub – to be inline with the brand 😁

Reply:
Good spot! Fixed by removing the whole file... because the doc is not actually needed.